README.md

```diff
@@ -3,136 +3,29 @@
- - [x] Supports the building of node applications.
- - [ ] Partially supports build parameters (first phase):
-    - [ ] Supports build dependencies.
-    - [ ] Supports the copying of build results from other modules.
-    - [ ] Supports the build results from a different location.
-    - [x] Supports target platforms.
- - [x] Supports the generation of a default `Makefile` file.
- - [ ] Supports the generation of an `mtad.yaml` file from an `mta.yaml` file.
- - [ ] Supports the building of `XSA` and `CF` (Cloud Foundry) targets.
-
-### Milestone 2 - (Q2 - 2019)
-
-  - [ ] Supports the generation of verbose `Makefile` files.
-  - [ ] Supports MTA extensions.
-  - [ ] Supports the building of Java and Maven applications.
-  - [ ] Supports ZIP builds.
-  - [ ] Supports fetcher builds.
-  - [ ] Supports build parameters:
-    - [ ] Supports build options.
-    - [ ] Supports `ignore` files and folders.
-    - [ ] Supports the definition of timeouts.
-    - [ ] Supports the naming of build artifacts.
-  - [ ] Supports multi-schema.
-  - [ ] Supports the enhancing of schema validations.
-  - [ ] Supports semantic validations.
-  - [ ] Partially supports the advanced `mta.yaml` (3.1 > 3.2) schema.
-
-
- ### Milestone 3 - (Q3 - 2019)
-
-  - [ ] Supports parallel execution for the default `Makefile` file.
-  - [ ] Supports incremental builds; in other words, one module at a time.
-
- ### Milestone 4 - (Q3 - 2019)
-
- - [ ] Supports the extensibility framework.
- - [ ] Fully supports the advanced `mta.yaml` (3.1 > 3.2) schema.
-
-## Download and Installation
-
- There are two supported ways to install the multi-target application archive builder (MBT) tool:
+#### The Cloud MTA Build Tool overview
+The Cloud MTA Build Tool is a standalone command-line tool that builds a deployment-ready
+multi-target application (MTA) archive `.mtar` file from the artifacts of an MTA project according to the project’s MTA
+development descriptor (`mta.yaml` file) or from module build artifacts according to the MTA deployment descriptor (`mtad.yaml` file). Also it provides commands for runing intermediate build process steps, for example, the mta.yaml file validations,building a single module according to configurations in the development descriptor, generating deployment descriptor, etc.
 
- **Manually**:
-  1. [Download](https://github.com/SAP/cloud-mta-build-tool/releases) the **latest** binary file according to your operating system.
-  2. Extract the archive file to the folder where you want to install the tool.
-  3. Add the binary file to your `~/bin` path according to your operating system:     
-        - Darwin / Linux
-          - Copy the binary file to the `~/usr/local/bin/` folder, for example: `cp mbt /usr/local/bin/`
-        - Windows
-          -  Copy the binary file `mbt.exe` to the `C:/Windows/` folder.
 
-**Use npm**:
-```
-npm install -g mbt
-```
+><b>For more information, see the the [Cloud MTA Build Tool user guide](docs/docs/index.md).</b>
 
```

Comment:
extra closing > at the beginning of the sentence.

Reply:
Sorry I just noticed there's an extraneous "the" in the above sentence.

Reply:
'>' is used to make the whole line to be colored as a note to attract attention.

Reply:
Double "the" fixed

Reply:
Thanks